collections/_version2-english/9-fork/2-to-dos.md

```diff
@@ -24,12 +24,22 @@ title: To-do's
 If you are a REP holder, there are some things you **MUST** do for a [fork]({{glossary_fork}}) to not lose your [REP]({{glossary_reputation_token}}).
 
 ## Before a fork starts
-REP holders have 2 things to do even before a [fork]({{glossary_fork}}) starts so that they don't lose the value of their [REP]({{glossary_reputation_token}}). One is to [convert REPv1 to REPv2](#convert-repv1-to-repv2) and the other is to [check monthly to see if a fork occurs](#check-monthly-to-see-if-a-fork-occurs).
+REP holders have a thing to do even before a [fork]({{glossary_fork}}) starts so that they don't lose the value of their [REP]({{glossary_reputation_token}}). That is to **check monthly to see if a fork occurs**.
```

Comment:
```suggestion
Before a [fork]({{glossary_fork}}) starts, REP holders still have to be actively involved so that they don't lose the value of their [REP]({{glossary_reputation_token}}). The minimum involvement necessary to avoid permanent losses of REP is to **check monthly to see if a fork occurs**.
```

Reply:
Thanks! I'll fix it tomorrow.

Reply:
I merged this suggestion. so I'm closing this conversation.